app/src/components/market/steps/items/create_market_step.tsx

```diff
@@ -104,33 +106,24 @@ const CreateMarketStep = (props: Props) => {
   const resolutionDate = resolution && formatDate(resolution)
 
   const buttonText = account ? 'Create' : 'Connect Wallet'
+  const balance = useSelector((state: any) => state.balance && new BigNumber(state.balance))
```

Comment:
@hpmaxi `balance` is going to be `any` here, right? Can we do better than that?

Reply:
I added some types to that selector